src/constants/index.ts

```diff
@@ -49,6 +49,7 @@ export const API = {
   checkAccessForAsset: (id: number): string => `/items/${id}/access`,
   checkFreeTrial: (id: number): string => `/items/used-trial-period/${id}`,
   getPackage: (id: number): string => `/items/packages/${id}`,
+  getAssetsInPackage: (id: number): string => `items/packages/${id}/items`,
```

Comment:
There was a discussion that we need to use v2 endpoints for the new stuffs. @FilipNikolovski 
But I am not sure if there is a v2 endpoint for listing the items in package.

Reply:
I'm not sure either, if there was a `v2` endpoint I would expect it to be named with `bundles` instead of `packages`.
In any case, please take over the ownership of this branch and PR while I'm on vacation.

Reply:
I rechecked this to make sure we're going with the right endpoint.
Calling the same endpoint with `v2/` prefix returns a status 405 error.